.github/actions/test-khiops-install/action.yml

```diff
@@ -1,74 +1,54 @@
-        then
-          # Put the full path in windows since bash doesn't find ".cmd" files
-          # in the path even if they are there
-          modl_path=$(which MODL)
-          khiops_bin_dir=$(dirname "$modl_path")
-          KHIOPS_SCRIPT="$khiops_bin_dir/khiops.cmd"
-          KHIOPS_CC_SCRIPT="$khiops_bin_dir/khiops_coclustering.cmd"
-          echo "KHIOPS_SCRIPT=$KHIOPS_SCRIPT" >> "$GITHUB_ENV"
-          echo "KHIOPS_CC_SCRIPT=$KHIOPS_CC_SCRIPT" >> "$GITHUB_ENV"
-        else
-          echo "KHIOPS_SCRIPT=khiops" >> "$GITHUB_ENV"
-          echo "KHIOPS_CC_SCRIPT=khiops_coclustering" >> "$GITHUB_ENV"
-        fi
-    - name: Check Khiops installation
-      shell: bash
-      run: |
-        export KHIOPS_PROC_NUMBER=4 
         export OMPI_MCA_rmaps_base_oversubscribe=true
-        "$KHIOPS_SCRIPT" -s
-        "$KHIOPS_CC_SCRIPT" -s
-    - name: Run Khiops tests
+        export KHIOPS_PROC_NUMBER=4
+        khiops-env --env
+    - name: Check Khiops output
       shell: bash
       run: |
-        cd test/LearningTest/TestKhiops/Standard/Iris/
-        "$KHIOPS_SCRIPT" -b -i test.prm -e results/err.txt
-        cd -
-    - name: Run Khiops Coclustering tests
+        export OMPI_MCA_rmaps_base_oversubscribe=true
+        export KHIOPS_PROC_NUMBER=4
+        khiops -s
+        khiops -v &> output
+        LINE_NUMBER=$(wc -l < output)
+        if [ $LINE_NUMBER -gt 1 ] ;
+        then 
+          echo "::error::Unexpected output in khiops scripts"
+          false
+        fi
+    - name: Check Khiops coclustering output
       shell: bash
-      run: |
-        cd test/LearningTest/TestCoclustering/Standard/Iris/
-        "$KHIOPS_CC_SCRIPT" -b -i test.prm -e results/err.txt
-        cd -
-    - name: Check the test results
+      run: |-
+        khiops_coclustering -s
+        khiops_coclustering -v &> output
+        LINE_NUMBER=$(wc -l < output)
+        if [ $LINE_NUMBER -gt 1 ] ;
+        then 
+          echo "::error::Unexpected output in khiops_coclustering scripts"
+          false
+        fi
+    - name: Check process number
       shell: bash
       run: |-
-        touch test/LearningTest/TestCoclustering/Standard/Iris/results/time.log
-        touch test/LearningTest/TestKhiops/Standard/Iris/results/time.log
-        echo "Check test results"
-        cd test/LearningTestTool/py/
-        $PYTHON kht_test.py ../../LearningTest/TestKhiops/Standard/Iris check
-        $PYTHON kht_test.py ../../LearningTest/TestCoclustering/Standard/Iris check
-        $PYTHON kht_apply.py ../../LearningTest/TestKhiops/Standard/Iris errors | tee /tmp/khiops-log.txt
-        $PYTHON kht_apply.py ../../LearningTest/TestCoclustering/Standard/Iris errors | tee /tmp/coclustering-log.txt
-        if (grep -q error /tmp/khiops-log.txt || grep -q error /tmp/coclustering-log.txt); then
-          echo "::error::Errors in Khiops run"
+        export OMPI_MCA_rmaps_base_oversubscribe=true
+        export KHIOPS_PROC_NUMBER=4
```

Comment:
Add these 2 env vars to `env:` section (see comments on this topic above).

Reply:
done